laravel-sso-example/resources/views/login.blade.php

```diff
@@ -1,30 +1,45 @@
 {{ Html::style('css/styles.css'); }}
+<script src="https://cdn.jsdelivr.net/gh/google/code-prettify@master/loader/run_prettify.js"></script>
 
 @if ($profile)
+<div class="logged_in_nav">
```

Comment:
I wasn't able to view the logged in page bc the app failed at the callback route (line 37 of web.php) with just a generic internal server error: 
![CleanShot 2022-08-26 at 14 12 24](https://user-images.githubusercontent.com/47123183/186983985-7e4a190b-a4cf-4c05-9215-56d6b099c6e8.png)